pkgs/shells/powershell/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv, fetchgit, dotnet-sdk, autoPatchelfHook, fetchzip, libunwind, libuuid, icu, git, curl, cmake, cacert,
+  glibcLocales, makeWrapper, less, openssl }:
+
+let bootstrap = vsn: stdenv.mkDerivation rec {
+  name = "powershell-bootstrap-${version}";
+  version = vsn;
+
+  src = fetchzip {
+    url = "https://github.com/PowerShell/PowerShell/releases/download/v${version}/powershell-${version}-linux-x64.tar.gz";
+    sha256 = "000mmv5iblnmwydfdvg5izli3vpb6l14xy4qy3smcikpf0h87fhl";
+    stripRoot = false;
+  };
+
+  buildInputs = [ autoPatchelfHook stdenv.cc.cc ];
+
+  installPhase = ''
+  mkdir -p $out/bin
```

Comment:
Nitpick: We use 4 Spaces in shell scripts and 2 in Nix code as indentation. 
https://nixos.org/nixpkgs/manual/#sec-syntax